features/authn.feature

```diff
@@ -3,12 +3,34 @@ Feature: Authenticate with Conjur
   Background:
     Given I setup a keycloak authenticator
 
-  Scenario: Authenticate with OIDC state and code
-    When I retrieve the login url for OIDC authenticator "keycloak"
+  Scenario: Authenticate with OIDC code
+    When I retrieve the provider details for OIDC authenticator "keycloak"
     And I retrieve auth info for the OIDC provider with username: "alice" and password: "alice"
     And I run the code:
     """
     $conjur.authenticator_enable "authn-oidc", "keycloak"
     Conjur::API.authenticator_authenticate("authn-oidc", "keycloak", options: @auth_body)
     """
     Then the JSON should have "payload"
+
+  Scenario: Authenticate with OIDC code requesting unparsed result via GET method
+    When I retrieve the provider details for OIDC authenticator "keycloak"
+    And I retrieve auth info for the OIDC provider with username: "alice" and password: "alice"
+    And I run the code:
+    """
+    $conjur.authenticator_enable "authn-oidc", "keycloak"
+    Conjur::API.authenticator_authenticate_get("authn-oidc", "keycloak", options: @auth_body)
+    """
+    Then the response body contains: "payload"
+    And the response includes headers
+
+  Scenario: Authenticate with OIDC code requesting unparsed result via POST method
```

Comment:
Two scenarios relying on the updated Conjur OIDC API currently fail in CI since the functionality has yet to be merged. The tests pass when run locally against a Conjur build containing those updates.